modules/update-mco-process.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="mco-update-process_{context}"]
+= Machine Config Operator node updates
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and compute node. During this process, the MCO performs the following sequential actions in the cluster:
+
+. Cordon and drain the nodes
+
+. Update the operating system (OS) of each node
+
+. Reboot the nodes
+
+. Uncordon the nodes
```

Comment:
this section will make the user confuse, above steps: cordon/drain, os update, reboot, uncordon are the sequential actions performed on one node (rolling update).

Reply:
Could you clarify what the process looks like when the MCO updates a group of 3 nodes at a time? 

Does the MCO perform each step of that process for all three nodes before moving on to the next step? (for example, cordon/drain the three nodes, and then update the OS of the three nodes....).

Or does the process look entirely different?

Reply:
Trevor describes it in https://github.com/openshift/openshift-docs/pull/60091#discussion_r1212387405 

TL;DR is that at most `maxUnavailable` nodes from a pool are in process. Steps between nodes are not coordinated, and as soon as one node is ready to receive workloads again, another may start the process.

Reply:
by default, MCO will perform the rolling update one by one (`spec.maxUnavailable=1`) in different machine config pools. the actions on one node is like cordon > drain > update (config+os) > reboot > uncordon. it won't codorn all nodes at first. etc.